simcase-game/src/components/Submit.js

```diff
@@ -0,0 +1,56 @@
+/** @jsx jsx */
+import React from "react";
+import { jsx } from "@emotion/core";
+import { createPortal } from "react-dom";
+import Confirm from "./Confirm";
+
+const buttonStyle = {
```

Comment:
We have to set time to create presentation component that we can reuse and not overpopulate main components.